{
	"error": "Đã xảy ra lỗi.",
	"cannot_identify_user": "❌ Không thể xác định người dùng."
}
